docs/changelog.rst

```diff
@@ -2,6 +2,8 @@
 Changelog
 =========
 
+* :release:`0.15.0 <2018-10-27>`
+* :bug:`2905` Don't cleanup mediator task if ExpireLock is not processed. Could leadto stuck channels.
```

Comment:
Humm, this is not a good explanation. **Because** the mediator task was cleared the **ExpiredLock** message is not processed, not if the expired lock is not processed then the mediator task is cleared